docs/case_studies/index.rst

```diff
@@ -86,73 +98,85 @@ Comparison table
      - Week
      - Week
      - Week
+     - Week
      - Day
    * - **Decision horizon**
      - 52 weeks
      - 52 weeks
      - 52 weeks
      - 52 weeks
+     - 52 weeks
      - 5 days
    * - **Network nodes**
      - 28
      - 9
      - 28
      - 28
+     - 9
      - 0
    * - **Production pads**
      - 14
      - 4
      - 15
      - 14
+     - 4
      - 5
    * - **Production tanks**
      - N/A
      - N/A
      - N/A
      - N/A
+     - N/A
      - 14
    * - **Completions pads**
      - 3
      - 1
      - 4
      - 3
+     - 2
      - 1
    * - **External completions pads** [#]_
      - 1 (CP03)
      - 0
      - 0
      - 1 (CP03)
+     - 1 (CP02)
      - N/A
    * - **Disposal sites (SWD)**
      - 5
      - 2
      - 3
      - 5
      - 2
+     - 2
    * - **Disposal expansion allowed?** [#]_
      - Yes, for K03 and K05
      - No
      - No
      - Yes, for K03 and K05
+     - Yes
      - No
    * - **Storage sites**
      - 3
      - 1
      - 2
      - 3
+     - 3
```

Comment:
I believe there is only one storage site actually connected to the network (S02) - let me know if I'm missing something. S03 and S04 are defined in the list of storage sites but are not connected to any other node in the system and are not pictured in the diagram.
```suggestion
     - 1
```

Reply:
@melody-shellman S03 and S04 are not used in the workshop baseline scenario, but are included in the input file so they can be used in the beneficial reuse scenario which is based off of the baseline scenario. I added a clarifying note to the documentation.

Reply:
okay! that's clear. Looks good.